src/scalper.ts

```diff
@@ -711,6 +753,7 @@ export class Scalper {
           console.log(this.symbol, "Gamma Scalp Fill From Taker?!", message.size, message.market, message.price, message.takerClientId, message.timestamp);
         }
         if (gammaFills > netGamma * gammaCompleteThreshold) {
+          gammaFills = 0;
```

Comment:
make this variable name clear that it is amount filled not number of fills

Reply:
gammaFillQty